### PR TITLE
Live % resizable. Background & Button  widgets. UNFINISHED

### PR DESCRIPTION
**Pull request by Zekiah-A**
Still making,  i'll finish later

### CODE REVIEW

**Comment by Zekiah-A** [comment]
I may also make one that doesn't use %, but normal pixel size which scales.

**Comment by Zekiah-A** [comment]
sus

File: GUILIB/Widgets/Widget.cs [hunk 1]
```diff
@@ -29,11 +29,13 @@ public abstract class Widget
 
         public Rectangle widgetRectangle;
         public Color color;
+        public bool scales;
 
-        public Widget(Rectangle widgetRectangle, Color color)
+        public Widget(Rectangle widgetRectangle, Color color, bool scaled)
```
Comment (StarlkYT) [inline GUILIB/Widgets/Widget.cs:34]: "scaled", did you mean "scales"?

Comment (Zekiah-A) [reply GUILIB/Widgets/Widget.cs:34]: yeah

Comment (Zekiah-A) [reply GUILIB/Widgets/Widget.cs]: I've changed it now, i think i had it like this because assigning a variable with the same name is just a bit  odd lol

**Comment by StarlkYT** [comment]
BTW is it finished?

**Comment by Zekiah-A** [comment]
Well, it only works for a few things, but yeah

**Comment by Zekiah-A** [comment]
Just accept it so that it doesn't get behind and i can make more PR's for the other widdgets / fixes in  the future

**Pull request merged**